webapp/redux/middleware/utils.js

```diff
@@ -1,7 +1,3 @@
-const createFileName = (name) => name + getTimeStamp();
```

Comment:
watch out when you delete functions, these two are still used by other functions except from `downloadPythonResponse `